draft-ietf-quic-multipath.md

```diff
@@ -381,6 +381,12 @@ that address. After receiving packets from the
 client on a new path, if the server decides to use the new path,
 the server MUST perform path validation ({{Section 8.2 of QUIC-TRANSPORT}})
 unless it has previously validated that address.
+A server can also decide to not perform path validation if it wants to deny the new path,
+or not perform it immediately if, e.g., no connection ID is yet available.
+If an endpoint decides to deny the establishment of a new path,
+this should be considered as a path closure and therefore requires sending a
+PATH_ABANDON frame on another path for the associated Path ID used
+by the client as specified in section {{path-close}}.
 
```

Comment:
I think that we are close to agreement. My suggestion would be to write:

An endpoint that cannot accept a path immediately has two choices. It can
decide to simply wait, waiting for example for an appropriate connection
ID; or, it can decide to deny the establishment of the new path. 
If the endpoint decides to deny the establishment of the new path,
this should be considered as a path closure and therefore requires sending a
PATH_ABANDON frame on another path for the associated Path ID used
by the client as specified in section {{path-close}}.

Reply:
There are actually two different cases here (and I agree it would be better to note them explicitly rather than mixing everything together into one sentence):

1. An endpoint receives a path validation on a new path and for whatever reason it simply doesn't want to open a new path or at least on that path. In this case, there is no point in sending the path_response but you rather send a path_abandon on another active path immediately.
2. An endpoint receives a path validation but it can't  reply to it because it doesn't have a CID for that path ID. In this case there are two option: either immediately send the path_abandon (and the other end could try again with a different path ID), or wait for a while to see if a CID for that path ID arrives. Further down in the text we actually don't recommend this second option except if the new CID frame was sent n the same packet but processed in the wrong order.

I think your proposed text doesn't cover case 1 accordingly and actually we might only really need to cover case 1 as case 2 is discussed further below. But again, I agree we need to adapt the wording.